setup.py

```diff
@@ -22,30 +22,32 @@ def hidapi_src(platform):
 
 if sys.platform.startswith("linux"):
     modules = []
-    if "--without-libusb" in sys.argv:
-        sys.argv.remove("--without-libusb")
-        hidraw_module = "hid"
-    else:
+    if "--with-libusb" in sys.argv:
+        sys.argv.remove("--with-libusb")
         hidraw_module = "hidraw"
```

Comment:
How is this supposed to work when you removed `hidraw.pyx`?

Reply:
I honestly didn't know exactly how Cython works, and figured that I could simply reuse a single `.pyx` source.  And I was wrong...

_(The `.pyx` is used to generated a `.c` file, which is cached and must contain the appropriate `Pyinit_<module>` symbol.  This is why reusing a `.pyx` file for different extensions doesn't work: all extensions after the first one would be missing their `PyInit_<module>` symbols.)_

Fixed, please take another look.